utils/wasm-gen/src/memory_layout.rs

```diff
@@ -0,0 +1,63 @@
+// This file is part of Gear.
```

Comment:
why here? you can move it to `wasm` module. There are related to memory entities too